[\#1768 Issue](https://github.com/rear/rear/issues/1768) `closed`: Recovering big endien
========================================================================================

**Labels**: `support / question`, `fixed / solved / done`,
`special hardware or VM`

#### <img src="https://avatars.githubusercontent.com/u/2150704?u=6c822a879b0b3bb6b268cafcd3d65789700fb378&v=4" width="50">[spacf](https://github.com/spacf) opened issue at [2018-04-04 09:11](https://github.com/rear/rear/issues/1768):

Hello!  
I am trying to create a working iso bootable that allow me to perform a
complete system recover without any issues. Currently I am not able to
do it.  
My lpar is using a SAN lun to boot. After I run 'rear mkbackup' I do a
'dd' of the iso created by rear to the LUN. After I select it to boot
from inside SMS menu.  
note.: I am using LVM.  
[rear-restore.txt](https://github.com/rear/rear/files/1875019/rear-restore.txt)

Attached is the output of my config and also the steps and messages
during boot process.

Can someone give me some hints on what to change/include to allow to
make rear recover to avoid the error ...:  
2018-04-04 10:38:58 ERROR: Backup archive
'/rearbackup/ch13x777/backup.tar.gz' not found !

#### <img src="https://avatars.githubusercontent.com/u/2150704?u=6c822a879b0b3bb6b268cafcd3d65789700fb378&v=4" width="50">[spacf](https://github.com/spacf) commented at [2018-04-04 09:29](https://github.com/rear/rear/issues/1768#issuecomment-378538989):

note .: /rearbackup is mounted on a lv created on a different LUN from
boot LUN.  
\[root@ch13x777 ~\]\# pvs  
/var/run/lvm/lvmetad.socket: connect failed: No such file or directory  
WARNING: Failed to connect to lvmetad. Falling back to internal
scanning.  
PV VG Fmt Attr PSize PFree  
/dev/mapper/mpathap3 vg\_ch13x777 lvm2 a--u 49.50g 0  
/dev/mapper/mpathc datavg lvm2 a--u 400.00g 390.00g

#### <img src="https://avatars.githubusercontent.com/u/1788608?u=925fc54e2ce01551392622446ece427f51e2f0ce&v=4" width="50">[jsmeix](https://github.com/jsmeix) commented at [2018-04-04 09:55](https://github.com/rear/rear/issues/1768#issuecomment-378545756):

See  
[https://github.com/rear/rear/blob/master/.github/ISSUE\_TEMPLATE.md](https://github.com/rear/rear/blob/master/.github/ISSUE_TEMPLATE.md)

#### <img src="https://avatars.githubusercontent.com/u/12116358?u=1c5ba9dcee5ca3082f03029a7fbe647efd30eb49&v=4" width="50">[gozora](https://github.com/gozora) commented at [2018-04-04 10:34](https://github.com/rear/rear/issues/1768#issuecomment-378555821):

You are doing backup to local directory
(`BACKUP_URL="file:///rearbackup"`).  
In order to restore from such configuration, you need to have your
backup archive (*backup.tar.gz*) available in `/rearbackup/ch13x777`
directory prior running `rear recover`.

V.

#### <img src="https://avatars.githubusercontent.com/u/1788608?u=925fc54e2ce01551392622446ece427f51e2f0ce&v=4" width="50">[jsmeix](https://github.com/jsmeix) commented at [2018-04-04 12:48](https://github.com/rear/rear/issues/1768#issuecomment-378587893):

Very informative!

@gozora  
many thanks for your time that you digged into that issue!

------------------------------------------------------------------------

\[Export of Github issue for
[rear/rear](https://github.com/rear/rear).\]
